app/assets/javascripts/tagging.js

```diff
@@ -1,6 +1,15 @@
 function addTag(tagname, selector) {
 
   selector = selector || '#tagform';
+  if (tagname.indexOf("place") !== -1) {
+    place = tagname.split(":")[1];
+    place.replace("-", " ");
+
+    var confirm = confirm("This looks like a location. Is this full description of the location accurate?");
```

Comment:
OK, so we have a flow issue here -- think about this, we probably need to do the geocoding /first/ and then present the user with the attempt to determine the full location /after/ it returns, right? 

So if so, we ought to just initiate the geocode automatically. Then, the `confirm()` step ought to go in the onComplete -- that way we can say `Is "____________" an accurate description of the place?`